.github/workflows/application.yml

```diff
@@ -105,9 +105,14 @@ jobs:
                 run: mv composer.lock old-composer.lock
                 if: steps.check-old-composer-lock-existence.outputs.files_exists == 'true'
 
-            -
-                name: Install PHP dependencies
-                run: composer update --no-interaction --no-scripts
+            -   name: Install PHP dependencies
+                if: matrix.php != '8.2'
+                run: composer update --no-interaction
+                id: end-of-setup
+
+            -   name: Install PHP 8.2 dependencies
+                if: matrix.php == '8.2'
+                run: composer update --no-interaction --no-scripts --ignore-platform-req=php
```

Comment:
Why is the `--ignore-platform-req=php` flag necessary here?

Reply:
When I opened the Pr, some of the dependencies are still not compatible with PHP 8.2 (e.g. PHPSpec). But I can give it a try and see what happens now 😄

Reply:
Sadly, we need to keep it as some dependencies are still not 8.2-compatible 🖖